PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -630,7 +632,7 @@ void ScintillaEditView::setXmlLexer(LangType type)
 {
 	if (type == L_XML)
 	{
-        execute(SCI_SETLEXER, SCLEX_XML);
+		execute(SCI_SETILEXER, 0, reinterpret_cast<LPARAM>(CreateLexer("xml")));

```

Comment:
"xml" and strings below might be replaced again by some definition/constexpr from N++